Section trees improve readability (#2272)
umbraco/UmbracoDocs: Extending/Section-Trees/trees.md

:::note

> In V8 the `/config/trees.config` file has been removed.

Reviewer: ```suggestion
In Umbraco 8 the `/config/trees.config` file has been removed.
```

public class FavouriteThingsTreeController : TreeController
```

> The edit view in the example would now be loaded from the location: */App_Plugins/favouriteThings/backoffice/favouriteThingsAlias/edit.html*

Reviewer: ```suggestion
The edit view in the example would now be loaded from the location: `/App_Plugins/favouriteThings/backoffice/favouriteThingsAlias/edit.html`
```

    return root;
}
```

> The RoutePath should be in the format of: **section/treeAlias/method**. As our example controller uses the `PluginController` attribute, clicking the root node would now request **/App_Plugins/favouriteThing/backoffice/favouritistThingsAlias/overview.html**. If you are not using the `PluginController` attribute, then the request would be to **/umbraco/views/favouritistThingsAlias/overview.html**

Reviewer: ```suggestion
The RoutePath should be in the format of: **section/treeAlias/method**. As our example controller uses the `PluginController` attribute, clicking the root node would now request `/App_Plugins/favouriteThing/backoffice/favouritistThingsAlias/overview.html`. If you are not using the `PluginController` attribute, then the request would be to `/umbraco/views/favouritistThingsAlias/overview.html`.
```
